fixtures/cmr.uat.earthdata.nasa.gov-443/162670989255318096

```diff
@@ -0,0 +1,27 @@
+GET /search/collections.json?concept_id=C1233147317-EEDTESTT&page_size=2000&include_tags=true
```

Comment:
It looks like some of these fixtures are outdated (include_tags=true vs include_tags=harmony.has-eula)

Reply:
Thanks for catching that. Updated here: https://github.com/nasa/harmony/pull/51/commits/5667f0b9bd3f1fa54fc5782b10a661de804b332b